### ISSUE TITLE
[Feature Request] Make text selectable during generation.

### ISSUE THREAD

**Issue by openMolNike** [LostRuins/koboldcpp#759]
Make text selectable during generation.

English is not my native language. Sometimes I want to copy a word to paste into the translation tool, but while the text generation continues, I cannot select new text, the selection is constantly reset.

**Comment by aleksusklim** [comment]
Not sure how this would be fixed given the nature of progressive HTML rendering, but this is how you can cope with it:
- Quickly `Ctrl+A` and `Ctrl+C`, then paste into the Notepad and grab the word you need.
- With enabled SSE streaming – just `Abort` the generation whenever you need to copy something. Then continue `Generate More`, it should not give you any errors or slowdowns!

**Comment by LostRuins** [comment]
Yep, what @aleksusklim said. The incoming streaming text (yellow text) is constantly refreshed so keeping it selected won't work. But the old (white text) should be selectable. Otherwise, you can consider changing the streaming mode.
